llvm/test/CodeGen/AMDGPU/store-private.ll

```diff
@@ -16,6 +17,47 @@
 
 ; SI: buffer_store_byte
 define amdgpu_kernel void @store_i1(ptr addrspace(5) %out) {
+; EG-LABEL: store_i1:
```

Comment:
This one at least is pretty broken. It didn't generate the relevant checks and only the R600 lines.

We probably shouldn't have any generated tests with shared r600 and gcn run lines